requirements.txt

```diff
@@ -7,4 +7,6 @@ psutil                # for logging extra CPU-info
 colorhash             # for hashing a string into a color
 numpy                 # for computing median and other stats
 pytz                  # for timezone info
-tzlocal               # for figuring out the local timezone
\ No newline at end of file
+tzlocal               # for figuring out the local timezone
+pymysql               # for mysql database connections
```

Comment:
the last two requirements don't belong here. they can be added by the users if they do decide to use mysql or postgresql